bioconda_utils/cli.py

```diff
@@ -537,13 +537,13 @@ def dag(recipe_folder, config, packages="*", format='gml', hide_singletons=False
      help="""Skip updating/bumping packges that are already built with
      compatible pinnings in one of the given channels in addition to those
      listed in 'config'.""")
-@arg('--bump-only-python',
-     help="""Bump package build numbers even if the only applicable pinning
-     change is the python version. This is generally required unless you plan
-     on building everything.""")
 @arg('--skip-variants',
      nargs='*',
      help='Skip packages that use one of the given variant keys.')
+@arg('--max-bumps', type=int,
+     help='Maximum number of recipes that will be updated.')
+@arg('--no-leaves',
+     help='Only update recipes with dependent packages.')
```

Comment:
The idea here is to update packages in certain batches on `bulk`.
With `--no-leaves` we leave (no pun intended!) out packages that are not needed by others and initially only bump the "inner nodes" of the dependency graph.
With `--max-bumps` we can limit the number of recipes that will get bumped. That way we can get issue smaller PRs from `bulk` to `master` and as such synchronize both branches more quickly. This will be particularly helpful in big rebuilds like for the compiler updates.